src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/PublicationServiceTests.cs

```diff
@@ -836,7 +836,10 @@ await using (var context = InMemoryApplicationDbContext(contextId))
                 Assert.Equal(topic.Theme.Title, publicationViewModel.Theme.Title);
 
                 // Do an in depth check of the saved release
-                var createdPublication = await context.Publications.FindAsync(publicationViewModel.Id);
+                var createdPublication = await context.Publications
+                    .Include(p => p.Contact)
+                    .Include(p => p.Topic)
+                    .FirstAsync(p => p.Id == publicationViewModel.Id);
```

Comment:
Fixed - and I've added new `await using (var context = InMemoryApplicationDbContext(contextId))` blocks for the same scenario across all test cases in this file.

I had to add a couple of `.Include(p => p.Topic)` lines elsewhere, so we were being hit by caching in test cases across this file. But the tests otherwise passed so all looks fine.

I also took this opportunity to remove any other `FindAsync` calls in the file.